pinot-common/src/main/java/org/apache/pinot/common/utils/FileUploadDownloadClient.java

```diff
@@ -148,6 +153,23 @@ public static URI getRetrieveAllSegmentWithTableTypeHttpUri(String host, int por
         rawTableName + TYPE_DELIMITER + tableType));
   }
 
+  public static URI getStartReplaceSegmentsHttpURI(URI controllerURI, String rawTableName, String tableType)
```

Comment:
Added java docs. Also, this supports both http/https because we reuse http/https provided from `controllerURI`. I changed the function name to `getStartReplaceSegmentsURI` to avoid the confusion.